packages/gestalt/src/Popover/InternalPopover.tsx

```diff
@@ -18,7 +18,7 @@ type Props = {
   color?: Color;
   onKeyDown?: (arg1: { event: React.KeyboardEvent<HTMLElement> }) => void;
   id?: string;
-  idealDirection?: 'up' | 'right' | 'down' | 'left' | 'forceDown';
+  idealDirection?: 'up' | 'right' | 'down' | 'left' | 'forceDown' | 'forceRight';
```

Comment:
Would this be a better approach if taken from packages/gestalt/src/utils/positioningTypes.ts using MainDirections?